Character.h

```diff
@@ -9,66 +9,70 @@
 
 struct Character
 {
-    Character(int hp, int armor_, int attackDamage_ );
-    virtual ~Character() { }
-    
+    Character(int hp, int armor_, int attackDamage_);
+    virtual ~Character() = default;
```

Comment:
Did clang-format do this change or did you watch the Chapter 5 vids that cover making special member functions `= default;` ?

Reply:
yeah, I have finished most of the videos but the projects are a little behind. I also set up a warning in clang-tidy to catch this.